controllers/topics.js

```diff
@@ -44,3 +45,15 @@ exports.getCommentsByArticleId = (req, res, next) => {
     });
 };
 
+exports.postComment = (req, res, next) => {
+    const { article_id } = req.params;
+    const { username, body } = req.body;
+    addComment(article_id, username, body)
+    .then((addedComment) => {
+        res.status(201).send({addedComment})
```

Comment:
`addedComment` is fine for your parameter name - but should really be on the key of simply `comment` for the response body